library/ecp.c

```diff
@@ -767,6 +767,58 @@ int mbedtls_ecp_point_write_binary( const mbedtls_ecp_group *grp,
     return( ret );
 }
 
+#if defined(MBEDTLS_ECP_SHORT_WEIERSTRASS_ENABLED)
+static int mbedtls_ecp_sw_derive_y( const mbedtls_ecp_group *grp,
+                                    const mbedtls_mpi *X,
+                                    mbedtls_mpi *Y,
+                                    int parity_bit )
+{
+    /* y^2 = x^3 + ax + b            (where a = -3 for short weierstrass)
```

Comment:
This comment needs updating: we support a != -3. (The code seems to know that, but the comment doesn't match the code.)

Reply:
:rocket: